CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.12.2
+* Bug - Fixed a bug where agent fills up the disk when querying stats for stopped containers.
```

Comment:
I don't think this is accurate.  Would it be bad to repeat the entry from 1.12.1?  I think that's still the best description; even though the fix is different it's still the same effect.